pom.xml

```diff
@@ -88,6 +100,23 @@
       <optional>true</optional>
     </dependency>
 
+    <dependency>
+      <groupId>org.postgresql</groupId>
+      <artifactId>postgresql</artifactId>
+    </dependency>
+
+    <dependency>
+      <groupId>org.mapstruct</groupId>
+      <artifactId>mapstruct</artifactId>
```

Comment:
Where we use this dependency? Is it required by the code generation plugin?

Reply:
I've removed this dependency, thanks - it was really unused